options.go

```diff
@@ -611,6 +620,41 @@ func parseJwtIssuers(issuers []string, msgs []string) ([]jwtIssuer, []string) {
 	return parsedIssuers, msgs
 }
 
+func parseIPWhitelist(ipWhitelistStrs []string, msgs []string) ([]*net.IPNet, []string) {
+	parsedIPNets := make([]*net.IPNet, 0, len(ipWhitelistStrs))
+
+	for ipIndex, ipStr := range ipWhitelistStrs {
+		if !strings.ContainsRune(ipStr, '/') {
+			ip := net.ParseIP(ipStr)
+			if ip == nil {
+				msgs = append(msgs, fmt.Sprintf(
+					"ip_whitelist[%d] (%s) looks like a IP address, but could not be recognized", ipIndex, ipStr))
+				continue
+			}
+
+			// nolint:gocritic
```

Comment:
What does `gocritic` complain about here?

Reply:
It want's me to make this a switch statement, which I can't in a straightforward way.  I'm not sure why it wants me to.

Reply:
(Wait, maybe I can?)

Reply:
No yeah I don't think I can

Reply:
Okay I definitely can, it's weird, but I'm doing it anyways.